### DISCUSSION TITLE
Repository for e2e_setup_test

### DISCUSSION THREAD

**Discussion by monopauli** [cosmos/gaia#2342] [Q&A]
Hi everyone,

When trying to run the e2e test suite, docker crashes with the following error: 
`API error (404): pull access denied for cosmos/gaiad-e2e, repository does not exist or may require 'docker login': denied: requested access to the resource is denied`
I am logged in on ghcr.io via docker.  Does this repository exist or is pulling simply not allowed?

**Comment by mmulji-ic** [comment]
Hello @monopauli , we don't push the `cosmos/gaiad-e2e` image to a centralised repo.
You have to build locally, using `make docker-build-all`, before running the end to end tests.
